library/src/engine.cxx

```diff
@@ -143,32 +143,62 @@ void engine::loadPlugin(const std::string& pathOrName)
     }
     else
     {
-      std::string libName;
+      std::vector<std::string> searchPaths;
+
+      std::string envValue;
+      if (vtksys::SystemTools::GetEnv("F3D_PLUGINS_PATH", envValue))
+      {
+        // split path with OS separator (':' on Linux/macOS and ';' on Windows)
+#ifdef _WIN32
+        char sep = ';';
+#else
+        char sep = ':';
+#endif
+        vtksys::SystemTools::Split(envValue, searchPaths, sep);
```

Comment:
use filesystem instead ?

Reply:
It's not related to fs here, it's just a split of a string to a string list.